lib/api_repo.js

```diff
@@ -77,7 +77,7 @@ function ApiRepo(opts) {
   this.pkgPrefix = opts.pkgPrefix;
   this.overridePlugins = opts.overridePlugins;
   this.zipUrl = opts.zipUrl;
-  this.isGoogleApi = !!opts.isGoogleApi;
+  this.isGoogleApi = !!opts.buildCommonProtos;
```

Comment:
It's not the same -- `isGoogleApi` applies if `buildCommonProtos` or `zipUrl == GOOGLE_APIS_REPO_ZIP` (see next line).